content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -14,16 +14,16 @@ weight: 50
 <!-- overview -->
 
 A Job creates one or more Pods and will continue to retry execution of the Pods until a specified number of them successfully terminate.
-As pods successfully complete, the Job tracks the successful completions.  When a specified number
-of successful completions is reached, the task (ie, Job) is complete.  Deleting a Job will clean up
+As pods are successfully completed, the Job tracks the successful completions.  When a specified number
+of successful completions is reached, the task (i.e. Job) is complete.  Deleting a Job will clean up
 the Pods it created. Suspending a Job will delete its active Pods until the Job
 is resumed again.
 
 A simple case is to create one Job object in order to reliably run one Pod to completion.
 The Job object will start a new Pod if the first Pod fails or is deleted (for example
 due to a node hardware failure or a node reboot).
 
-You can also use a Job to run multiple Pods in parallel.
+You can also use a Job to run multiple Pods in parallel. See [this](https://kubernetes.io/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).
```

Comment:
We prefer to use `/docs/tasks/...` link instead of `https://kubernetes.io/docs/...`.
```suggestion
You can also use a Job to run multiple Pods in parallel. See [this](/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).
```

Reply:
@utkarsh-singh1 , we use reference paths for links.
@Sea-n 's comment is [correct](https://github.com/kubernetes/website/pull/34445/files#r903501103)